### ISSUE TITLE
Visualize the advantage / benefits to using DracProt on front page

### ISSUE THREAD

**Issue by rayraspberry** [Dracula-Protocol/frontend#13]
Current Behavior:  Landing/Main page shows dashboard for users

Issue:  Much of the information isn't relevant until the user connects a wallet.

Suggested behavior:  Main Landing page when the wallet is not connected should show people why they should be staking on DRC...  In the most simple and direct terms.

Visualize -- some sort of moving / ticking / spinning / numbers counting up for per $100 or $1000 dollars invested: 
Gas saved (estimate) -- 
Eth Earned by Average (or Median) LP staker - or DRC staker or whatever - 
Top pools APY.   

Let the # invested be on a slider so people can set it from $100 to $10,000  to see how much gas they could say and compounding ETH rewards they could earn based on criteria.

Maybe to start off the front page could be the v2 Diagram [ https://miro.medium.com/max/3200/0*KIeGFcgcZodTeef8 ]  but animated and instead of the percentages being prominent -- make it the dollar values (or make it a choice between currencies / ETH / BTC / etc) 

Basically the landing / front page before connecting the wallet should be an advertisement -- when a wallet is connected the dashboard can appear.

Attention is an asset and if people are only going to check out the page once - we need to pique their interest immediately with why DRC is a good option for them and why it will save them money
